lib/ndr_import/file/xml.rb

```diff
@@ -18,17 +18,25 @@ class Xml < Base
       def rows(&block)
         return enum_for(:rows) unless block
 
-        xpath = @options['xml_record_xpath']
-
         if @options['slurp']
-          doc = read_xml_file(@filename)
-          doc.xpath(xpath).each(&block)
+          record_elements(read_xml_file(@filename)).each(&block)
         else
+          xpath = "*/#{@options['xml_record_xpath']}"
```

Comment:
This doesn't look it supports `xml_record_xpath` as a regexp.

Reply:
No it doesn't at the moment, I'll look at adding support for it

Reply:
pattern matching on `xml_record_xpath` now supported when streaming 52738ff